pybaseball/utils.py

```diff
@@ -67,18 +67,18 @@ def sanitize_input(start_dt, end_dt, player_id):
     return start_dt, end_dt, player_id
 
 
-def split_request(start_dt, end_dt, player_id, url):
+def split_request(start_dt: str, end_dt: str, player_id: int, url: str) -> pd.DataFrame:
     """
     Splits Statcast queries to avoid request timeouts
     """
     current_dt = datetime.datetime.strptime(start_dt, '%Y-%m-%d')
-    end_dt = datetime.datetime.strptime(end_dt, '%Y-%m-%d')
+    end_dt_datetime = datetime.datetime.strptime(end_dt, '%Y-%m-%d')
```

Comment:
???

Reply:
MyPy flags this code as not using a best practice. It reuses an existing variable AND changes its type (from str to datetime). This var name change disambiguates it, so we're clear on which is the string and which is the datetime.

Reply:
why datetime and not date?

Reply:
We're on the same page. Just fixed all datetimes to dates.